docs/source/deployment/deploy-model-to-kubernetes/tutorial.rst

```diff
@@ -0,0 +1,487 @@
+    :align: center
+    :figwidth: 80%
+
+In this example, the best model appears to be in the top-left corner, with ``alpha=0.2`` and ``l1_ratio=0`` (you may see different results).
+
+Step 5: Packaging the Model and Dependencies
+--------------------------------------------
+Since we are using autologging, Mlflow automatically logsthe `Model <../../../models.html>`_ for each run. This process conveniently packages the model weight
+and dependencies in a ready-to-deploy format.
+
+.. note::
+
+  In practice, it is also recommended to use `MLflow Model Registry <../../../model-registry.html>`_ for registering and managing your models.
+
+
+Let's take a brief look at how this format appears. You can view the logged model through the ``Artifacts`` tab on the Run detail page.
+
+.. code-block::
+
+  model
+  ├── MLmodel
+  ├── model.pkl
+  ├── conda.yaml
+  ├── python_env.yaml
+  └── requirements.txt
+
+``model.pkl`` is the file containing the serialized model weight. ``MLmodel`` includes general metadata that instructs MLflow on how to load the model.
+The other files specify the dependencies required to run the model.
+
+.. note::
+
+  If you opt for manual logging, you will need to log the model explicitly using the :py:func:`mlflow.sklearn.log_model <mlflow.sklearn.log_model>`
+  function, as shown below:
+
+  .. code-block:: python
+
+    mlflow.sklearn.log_model(lr, "model")
+
+.. _step-6-test-model-serving-locally:
+
+Step 6: Testing Model Serving Locally
+-------------------------------------
+
+Now that you are ready to deploy the model, but let's first test the model serving locally. As outlined in the
+`Deploy MLflow Model Locally <../deploy-model-locally.html>`_, you can run a local inferecen server with just a single command.
+Remember to use the ``enable-mlserver`` flag, which instructs MLflow to use MLServer as the inference server. This ensures the model runs in the
+same manner as it would in Kubernetes.
+
+.. code-block:: bash
+
+  mlflow models serve -m runs:/<run_id_for_your_best_run>/model -p 1234 --enable-mlserver
+
+This command starts a local server listening on port 1234. You can send a request to the server using ``curl`` command:
+
+.. code-block:: bash
+
+    $ curl -X POST -H "Content-Type:application/json" --data '{"inputs": [[14.23, 1.71, 2.43, 15.6, 127.0, 2.8, 3.06, 0.28, 2.29, 5.64, 1.04, 3.92, 1065.0]]' http://127.0.0.1:1234/invocations
+
+    {"predictions": [-0.03416275504140387]}
+
+For more information about the request format and response formats, refer to :ref:`Inference Server Specification <local-inference-server-spec>`.
+
+
+Step 7: Deploying the Model to KServe
+-------------------------------------
+
+Finally we are all set to deploy the model to the Kubernetes cluster.
+
+Create Namespace
+~~~~~~~~~~~~~~~~
+
+First, create a test namespace for deploying KServe resources and your model:
+
+.. code-block:: bash
+
+  $ kubectl create namespace mlflow-kserve-test
```

Comment:
```suggestion
  kubectl create namespace mlflow-kserve-test
```

because we don't use `$` in other code blocks.

Reply:
True, will remove remaining ones as well